server/db/models/order.js

```diff
@@ -0,0 +1,33 @@
+const Sequelize = require('sequelize');
+const db = require('./database');
+
```

Comment:
Order model only needs status and total cost because Sequelize magic methods will take care of the rest